chromium_src/chrome/browser/browser_process_impl.cc

```diff
@@ -3,7 +3,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "extensions/buildflags/buildflags.h"
+
+#if BUILDFLAG(ENABLE_EXTENSIONS)
 #include "brave/browser/extensions/brave_extensions_browser_client_impl.h"
+#endif
```

Comment:
this buildflag also needs to wrap the `define` below https://github.com/brave/brave-core/pull/4374/files#diff-e4ca40b9bcd69217b5335621fd53b90dR12 or it will break because `BraveExtensionsBrowserClientImpl` will be undefined

Reply:
It doesn't currently break because `ChromeExtensionsBrowserClient` is only used in the overridden file under the same buildflag, but it's definitely better to include the redefinition under the flag. Fixed in 2c0ec4bd68e3d5bcbcafe07f751d55f1b70f42bb